pkg/cmd/registry/registrycmdutil/flagset.go

```diff
@@ -0,0 +1,35 @@
+package registrycmdutil
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/spf13/cobra"
+)
+
+type FlagSet struct {
+	cmd     *cobra.Command
+	factory *factory.Factory
+	*flagutil.FlagSet
+}
+
+// NewFlagSet returns a new flag set with common Service Registry flags
+func NewFlagSet(cmd *cobra.Command, f *factory.Factory) *FlagSet {
+	return &FlagSet{
+		cmd:     cmd,
+		factory: f,
+		FlagSet: flagutil.NewFlagSet(cmd, f.Localizer),
+	}
+}
+
+// AddRegistryInstance adds a flag for setting the Service Registry instance ID
```

Comment:
Todo: an issue after it gets merged to have it across all registry commands.

Reply:
I would say if we keep feature branch we can avoid creating issues. Feature branch will allow us to resolve all issues/technical debt.